src/pages/popup/Popup.tsx

```diff
@@ -9,6 +9,15 @@ import withErrorBoundary from '@src/shared/hoc/withErrorBoundary';
 const Popup = () => {
   const theme = useStorage(exampleThemeStorage);
 
+  const injectContentScript = async () => {
+    const [tab] = await chrome.tabs.query({ currentWindow: true, active: true });
+    console.log(tab.id)
```

Comment:
This is for test i think
Don't forget to remove it :)

Reply:
Thank you for the double check.

Reply:
I deleted `conosle.log`